README.md

```diff
@@ -628,218 +623,3 @@ rules: [] # The control plane automatically fills in the rules
 # Extensions
```

Comment:
Let's remove the entire Extensions section.

Reply:
Done. 1c0356458a54f4c896a6de8674f77995b0472a8c